src/include/sof/platform.h

```diff
@@ -54,6 +54,24 @@ int platform_context_save(struct sof *sof);
 
 /** @}*/
 
```

Comment:
@lgirdwood @plbossart @lyakh can you please review as it touches the core. 

Our hardware (i.MX8ULP) has an interesting mapping for some shared memory.

DSP physical address range:

```0x1a000000~0x1bffffff```

is mapped to ARM physical address range:
```
0x8e000000~0x8fffffff 
```
so we need somehow to do a `translation` of the addresses.